### ISSUE TITLE
Super Events - End time of the Event schedule is missing

### ISSUE THREAD

**Issue by Priyaravi8** [rokwire/illinois-app#32]
**Describe the bug**
For the event "Discover healthy Aging Series", the end time is missing for Event Schedule list

**To Reproduce**

1. Install Illinois app
2. Complete Onboarding steps
3. Log in as "University student"
4. Tap on Events on the Home screen. Search for "Discover healthy Aging Series"
5. Discover healthy Aging Series Event is displayed with sub-events

**Actual Result**
On the Event schedule section, End time of the session is missing

**Expected behavior**
Display the End time of the session. If not remove the "-" from "10AM-"

**Screenshots**
![End time of event](https://user-images.githubusercontent.com/26231179/91896960-401e5780-ec5f-11ea-970b-7907cb7a8da0.png)

**Smartphone (please complete the following information):**
 - Device: [e.g. Android]
  - Version [e.g. 2.2.15]

**Comment by dobromirdobrev** [comment]
@Priyaravi8 , this is not a client issue. There is no `endDate` in the data that comes from server cc @sandeep-ps .

Here is the json response for these events:

```
[
  {
    "allDay": false,
    "category": "Community",
    "contacts": [
      
    ],
    "cost": "",
    "createdBy": "courtney@illinois.edu",
    "id": "5f4d5a1f507e9400096b809a",
    "imageURL": "https://api.rokwire.illinois.edu/events/5f4d5993d16aafb363e7686f/images/5f4d5a1fd16aafb363e76876",
    "isSuperEvent": false,
    "location": {
      "description": "Online"
    },
    "longDescription": "What is the secret to successful aging? In the first webinar of four, participants explore the characteristics of positive aging through discussion on ageism and the many surprising similarities of older and younger generations. Through these discussions, Family Life Team Educators Cheri Burcham and Tessa Hobbs Curley aim to stress the importance of positivity infused lifestyle choices, friendships and attitudes in healthy aging.",
    "name": "",
    "sponsor": "University of Illinois Extension",
    "startDate": "Thu, 03 Sep 2020 15:00:00 GMT",
    "subEvents": [
      
    ],
    "subcategory": "",
    "tags": [
      "Aging",
      "Mental Health"
    ],
    "targetAudience": [
      
    ],
    "title": "I\u2019m Positive, I\u2019m Aging",
    "titleURL": "http://go.illinois.edu/discoveraging",
    "rokwire_score": 1.29
  },
  {
    "allDay": false,
    "category": "Community",
    "contacts": [
      
    ],
    "cost": "Free",
    "createdBy": "courtney@illinois.edu",
    "id": "5f4d5fb5d389640009327603",
    "imageURL": "https://api.rokwire.illinois.edu/events/5f4d5d67d16aafb363e7688c/images/5f4d5fb5d16aafb363e768a6",
    "isSuperEvent": false,
    "location": {
      "description": "Online"
    },
    "longDescription": "When is the last time you laughed a hearty, sidesplitting belly laugh? According to this article from Mayo Clinic, humor has both physical and psychological health benefits. Think of the last time you laughed so hard it hurt - it\u2019s medicine for the soul. Family Life Team Educators Tessa Hobbs Curley and Chelsey Byers will demonstrate ways to use humor to cope with stress and changes in life. ",
    "name": "",
    "sponsor": "University of Illinois Extension",
    "startDate": "Thu, 10 Sep 2020 15:00:00 GMT",
    "subEvents": [
      
    ],
    "subcategory": "",
    "tags": [
      "Aging",
      "Mental Health"
    ],
    "targetAudience": [
      
    ],
    "title": "Looking for the Funny Side of Life",
    "titleURL": "http://go.illinois.edu/discoveraging",
    "rokwire_score": 1.29
  },
  {
    "allDay": false,
    "category": "Community",
    "contacts": [
      
    ],
    "cost": "Free",
    "createdBy": "courtney@illinois.edu",
    "id": "5f4d5fc4d389640007192e18",
    "imageURL": "https://api.rokwire.illinois.edu/events/5f4d5e47d16aafb363e76891/images/5f4d5fc4d16aafb363e768ad",
    "isSuperEvent": false,
    "location": {
      "description": "Online"
    },
    "longDescription": "Most everyone has days, or seasons of life, where it\u2019s hard to find motivation to accomplish tasks. Will Rogers once said, \"Even if you're on the right track, you'll get run over if you just sit there.\" Family Life Team Educators Karla Belzer and Molly Hofer will help listeners hone in on what causes those motivational ups and downs, and give ideas on how to boost enthusiasm and increase productivity. ",
    "name": "",
    "sponsor": "University of Illinois Extension",
    "startDate": "Thu, 17 Sep 2020 15:00:00 GMT",
    "subEvents": [
      
    ],
    "subcategory": "",
    "tags": [
      "Aging",
      "Mental Health"
    ],
    "targetAudience": [
      
    ],
    "title": "Finding Your Get Up and Go",
    "titleURL": "http://go.illinois.edu/discoveraging",
    "rokwire_score": 1.29
  },
  {
    "allDay": false,
    "category": "Community",
    "contacts": [
      
    ],
    "cost": "Free",
    "createdBy": "courtney@illinois.edu",
    "id": "5f4d5fd0d389640009327604",
    "imageURL": "https://api.rokwire.illinois.edu/events/5f4d5f87d16aafb363e76896/images/5f4d5fd0d16aafb363e768b4",
    "isSuperEvent": false,
    "location": {
      "description": "Online"
    },
    "longDescription": "The Healthy Aging series wraps up with a tactile way to encourage living life to the fullest by addressing the \u201cone of these days,\u201d \u201cWhen things slow down,\u201d and dreams on the back burner while \u201clife happens.\u201d  A bucket list is simply a list of things you\u2019d like to do in your lifetime. Simple in theory, but the point is often missed. Family Life Team Educators Chelsey Beyers and Karla Belzer challenge you to actually put those big goals and dreams down on paper and actively pursue your interests live your life to the fullest, one bucket list item at a time.",
    "name": "",
    "sponsor": "University of Illinois Extension",
    "startDate": "Thu, 24 Sep 2020 15:00:00 GMT",
    "subEvents": [
      
    ],
    "subcategory": "",
    "tags": [
      "Aging",
      "Mental Health"
    ],
    "targetAudience": [
      
    ],
    "title": "Someday is Today - Live Your Bucket List",
    "titleURL": "https://go.illinois.edu/discoveraging",
    "rokwire_score": 1.29
  }
]
```

**Comment by sandeep-ps** [comment]
@dobromirdobrev, yes, endDate is not a required field. So, I think if it is not set in the incoming data, the hyphen needs to be removed. In such a case, it'll be better to display the event time as `Thursday, 10 AM`, instead of `Thursday, 10 AM-` for example.

**Comment by dobromirdobrev** [comment]
Fixed for 2.3.0 and above. Now the client won't show the hyphen if there is no `endDate` value.

**Comment by Priyaravi8** [comment]
Rested this issue. This is fixed.
